inst/@double/harmonic.m

```diff
@@ -0,0 +1,72 @@
+%% Copyright (C) 2017 Colin B. Macdonald
+%%
+%% This file is part of OctSymPy.
+%%
+%% OctSymPy is free software; you can redistribute it and/or modify
+%% it under the terms of the GNU General Public License as published
+%% by the Free Software Foundation; either version 3 of the License,
+%% or (at your option) any later version.
+%%
+%% This software is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty
+%% of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+%% the GNU General Public License for more details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @documentencoding UTF-8
+%% @defun harmonic (@var{x})
+%% Numerical harmonic function.
+%%
+%% Example:
+%% @example
+%% @group
+%% harmonic (1.1)
+%%   @result{} ans = 1.0626
+%% @end group
+%% @end example
+%%
+%% @strong{Note} this function may be slow for large numbers of inputs.
+%% This is because it is not a native double-precision implementation
+%% but rather the numerical evaluation of the SymPy function
+%% @code{harmonic}.
+%%
+%% Note: this file is autogenerated: if you want to edit it, you might
+%% want to make changes to 'generate_functions.py' instead.
+%%
+%% @seealso{@@sym/harmonic}
+%% @end defun
+
+
+function y = harmonic (x)
+  if (nargin ~= 1)
+    print_usage ();
+  end
+  cmd = { 'L = _ins[0]'
+          'A = [complex(harmonic(complex(x))) for x in L]'
+          'return A,' };
+  c = python_cmd (cmd, num2cell(x(:)));
+  assert (numel (c) == numel (x))
+  y = x;
+  for i = 1:numel (c)
+    y(i) = c{i};
+  end
```

Comment:
Yes, that does look poor.  Outputs here are doubles (not sym) but I think `cell2mat` will help.